content/sample/images.md

```diff
@@ -1,26 +1,173 @@
 ---
-title: Responsive Images
-sub: Using nunjucks
+title: Image Layout Helpers
+sub: Using nunjucks and the image utility classes
 ---
 
 {% import 'embed.macros.njk' as embed %}
 
-## srcSet
 
-### embed (no width):
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/graph.jpg',
+    alt: 'sketched graph on paper'
+  }],
+  class='media-pull',
+  attrs={'style': '--rowspan: span 2'}
+) }}
+
+### `media-pull` will position an image next to the text on wide enough screens
+
+The `media-pull` class can be used to pull an image to the left of the content
+at the `xlarge` screen size. Using a style attribute, you can define how many
+rows to span using the css custom property `--rowspan`. The first image example
+uses the `embed.figure` macro.
+
 {{ embed.img(
-  src='blog/2016/trick-loops.jpg'
+  src='blog/2023/graph.jpg',
+  alt='A collection of button screenshots from one project',
+  attrs={
+    'class': 'media-pull',
+    'style': '--rowspan: span 2'
+  }
 ) }}
 
-### embed (width: 4410):
+### `media-pull` with the `embed.img` macro
+
+When using the `embed.img` macro in a markdown file, the image is placed
+within a `<p>` tag. We have a CSS helper that targets this nested image with
+`display: contents` in order to add layout classes which will work in [most
+modern browsers](https://caniuse.com/css-has). When unsupported, the image will
+appear centered within the content column.
+
```

Comment:
@mirisuzanne I added this second example specifically for testing the embed.img macro since it doesn't work in Firefox yet.